cli/src/aws_sso_util/populate_profiles.py

```diff
@@ -149,6 +149,26 @@ def trim_formatter(i, n, **kwargs):
         return formatter(i, n, **kwargs)
     return trim_formatter
 
+def get_name_case_formatter(account_name_transform, role_name_transform, formatter):
+    field_transform_map = {
+        "account_name": account_name_transform,
+        "role_name": role_name_transform,
+    }
+    def case_formatter(i, n, **kwargs):
+        for field, transform in field_transform_map.items():
+            if transform == "capitalize":
```

Comment:
This needs a null guard, in case only one argument is provided.
```
if not transform:
    continue
```
error:
```
aws-sso-util configure populate --dry-run --region us-east-2 --role-name-case lower
Usage: aws-sso-util configure populate [OPTIONS]

Error: Invalid profile name format: Unknown account_name transform value None
```

Reply:
Ooh, I saw that in the trim options and couldn't see why it was there. I assumed Click would protect us from that situation, but invoking the function for both options catches us out. Added in the most recent commit. :^)